tests/unit/util/CMakeLists.txt

```diff
@@ -32,12 +32,15 @@ blt_add_test(
   NAME allocation_map_tests
   COMMAND allocation_map_tests)
 
-blt_add_executable(
-  NAME allocation_map_performance_tests
-  SOURCES allocation_map_performance_tests.cpp
-  DEPENDS_ON umpire_util gtest
-  OUTPUT_DIR ${UMPIRE_TEST_OUTPUT_DIR})
+# TODO: Make this test a relative performance test
```

Comment:
Can we only disable this when running coverage tests?

Reply:
That was previously done - it didn't help.